mods/d2k/rules/defaults.yaml

```diff
@@ -131,13 +123,16 @@
 	TargetableUnit:
 		TargetTypes: Ground
 		RequiresForceFire: yes
-	Capturable:
-		Type: husk
-		AllowAllies: yes
-		CaptureThreshold: 1.0
-	TransformOnCapture:
-		ForceHealthPercentage: 25
+#	Capturable:
+#		Type: husk
+#		AllowAllies: yes
+#		CaptureThreshold: 1.0
+#	TransformOnCapture:
+#		ForceHealthPercentage: 25
```

Comment:
We probably want to keep this in to avoid confusion (since this is available in the other mods).